backend/routes/Auth.js

```diff
@@ -0,0 +1,78 @@
+import express from "express"
+import bcrypt from "bcryptjs"
+import jwt from "jsonwebtoken"
+import User from "../model/User.js"
+import authenticateUser from "../middleware/authenticateUser.js"
+import Thought from "../model/Thought.js"
+
+const authRouter = express.Router()
+
+authRouter.post("/signup", async (req, res) => {
```

Comment:
Endpoints are most often named nouns in plural, so instead of signup and login, it could be POST /users and POST /sessions